modules/interfaces/src/functional-group.ts

```diff
@@ -19,11 +26,23 @@ export type UserDefinitions = {
   [EntityName: string]: UserDefinition;
```

Comment:
```suggestion
  [AuthEntityName: string]: UserDefinition;
```
Suggestion to use matching name so that user will have an idea of what `AN` stands for

Reply:
I agree with your idea. I've come to think that `UserEntityName` would be more direct and easier to understand than `AuthEntityName`. So I'd like to replace `AuthEntity` with `UserEntity` and `AN` with `UN` at the next commit.

Reply:
Sounds good!

Reply:
Replacing these types was unexpectedly tough. I'd like to make another PR for this change.
Therefore, your suggestion will be resolved there and this PR can be merged immediately.
Can I merge this first?

Reply:
Sure, let's merge this first!